doc/changes/changes_12.0.1.md

```diff
@@ -0,0 +1,50 @@
+# Virtual Schema Common JDBC 12.0.1, released 2025-03-12
+
+Code name: Security Policy
+
+## Summary
+
+In this release we added a security policy that explains how to best report vulnerabilities.
+We also removed the deprecated `EXCEPTION_HANDLING` property.
+
+## Features
+
+* 159: Added security policy
```

Comment:
Check with `mvn com.exasol:project-keeper-maven-plugin:verify-release` if referencing a PR here works.

As an alternative move the link to the end:
```suggestion
* Added security policy (PR #159)
```

Reply:
Changed.